apmapf.py

```diff
@@ -165,16 +230,26 @@ def run(self):
         H_MIN = max(distance_to_target)  # horizon
         H_MAX = int(H_MIN * self.h_mult)
         H_MAX_ORIG = H_MAX
-        logging.log(logging.INFO, 'BFS lower bound: {}.'.format(H_MIN))
-        logging.log(logging.INFO, 'Setting H_MAX to h_mult * {}: {}'.format(H_MIN, H_MAX))
+
+        logging.log(logging.INFO, 
+                    'BFS lower bound: {}.'.format(H_MIN))
+
+        logging.log(logging.INFO, 
+                    'Setting H_MAX to h_mult * {}: {}'.format(H_MIN, H_MAX))
+
         if self.search == 'binary':
             H = (H_MAX - H_MIN) // 2 + H_MIN
         elif self.search == 'linear':
             H = H_MIN
         else:
-            logging.log(logging.ERROR, 'Search method is not one of "linear", "binary".')
+            logging.log(logging.ERROR, 
+                        'Search method is not one of "linear", "binary".')
             exit()
+        
         logging.log(logging.INFO, 'Beginning search with horizon: {}'.format(H))
+        
+        # H denotes the complexity we have checked 'up to', and H_MAX the
```

Comment:
Was not 100% sure about this.  (Maybe there is a better word here than just "complexity"?)

Reply:
`H` is the current horizon -- it determines the makespan we are currently checking. E.g. if `H` is too small, there will be no solution and the solver returns unsat.